### PR TITLE
solid -> op in versioning error

### PR DESCRIPTION
**Pull request by sryza**
## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by dpeng817 — CHANGES_REQUESTED** [review]
This should be using the `node_type_str` property so that it can properly switch between the two

File: python_modules/dagster/dagster/core/execution/resolve_versions.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def resolve_step_versions(pipeline_def, execution_plan, resolved_run_config):
         for input_name, version in input_version_dict.items():
             if version is None:
                 raise DagsterInvariantViolationError(
-                    f"Received None version for input {input_name} to solid {solid_def.name}."
+                    f"Received None version for input {input_name} to op {solid_def.name}."
```
Comment (dpeng817) [inline python_modules/dagster/dagster/core/execution/resolve_versions.py:94]: here too 🥺

**Review by dpeng817 — CHANGES_REQUESTED** [review]
This should be using the `node_type_str` property so that it can properly switch between the two

File: python_modules/dagster/dagster/core/execution/resolve_versions.py (hunk 1 above)
Comment (sryza) [reply python_modules/dagster/dagster/core/execution/resolve_versions.py:94]: oops - thanks

**Review by dpeng817 — APPROVED** [review]

**Pull request merged**